### PR TITLE
Account activate

### PR DESCRIPTION
**Pull request by hunguyen1702**
---
This change is [Reviewable](https://reviewable.io/reviews/framgia/fels_nvh/11)

### CODE REVIEW

**Comment by hunguyen1702** [comment]
ready

File: app/views/user_mailer/account_activation.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<%= content_tag :h1, t("title") %>
+
+<%= content_tag :p,
+  t("mailer.account_activation.greeting", {user_name: @user.name}) %>
+
+<%= content_tag :p, t("mailer.account_activation.content") %>
+
+<%= link_to t("mailer.account_activation.activate"),
+  edit_account_activation_url(@user.activation_token, email: @user.email) %>
+
```
Comment (hungnh103) [inline app/views/user_mailer/account_activation.html.erb]: xóa dòng

File: app/views/user_mailer/account_activation.text.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Hi <%= @user.name %>,
```
Comment (hungnh103) [inline app/views/user_mailer/account_activation.text.erb]: I18n

**Comment by hungnh103** [comment]
merged

**Pull request merged**